### PR TITLE
Clear recently used CRS button added - issue #8216

### LINKED ISSUE #8216: Add clear recently used crs button (Request in QGIS)
### Request for documentation
From pull request QGIS/qgis#52321
Author: @YoannQDQ
QGIS version: 3.32

**Add clear recently used crs button**

### PR Description:
Fixes #21473

## Description

- Adds a column in the recent crs table with a button to clear individual CRS
- Adds a context menu (right click) with two actions:
  - Clear selected (Same as button)
  - Clear all recent CRS (prompt user for confirmation)
- Pressing the Del key while the list of recently used crs has focus remove the selected CRS

clear_crs

### Commits tagged with [need-docs] or [FEATURE]

### PR DESCRIPTION
**Pull request by selmaVH1**
fixes #8216 
Added image and description for Clear recently used CRS.

### CODE REVIEW

File: docs/user_manual/working_with_projections/working_with_projections.rst [hunk 1]
```diff
@@ -202,6 +202,21 @@ default.
...
+
+   Clear recently used CRS 
+
+If you want to clear recently used CRS that can be done in several ways:
+
+   * Go to :guilabel:`Remove from recently used CRS` button on the right side.
+   * Right-click on used CRS to open context menu, choose
+     :guilabel:`Remove selected CRS from recently used CRS` to remove selected CRS
+     or choose :guilabel:`Clear all recently used CRS` to remove all used CRS.
+   * Press :kbd:`Del` after selecting CRS that you want to remove.
```
Comment (DelazJ) [inline docs/user_manual/working_with_projections/working_with_projections.rst:218]: I don't think this is the most appropriate place to describe the feature. The next section "crs_selector" would better fit; it is dedicated to the dialog and there is a specific entry for recently used crs. Better move the whole bits there.

**Review by DelazJ — COMMENTED** [review]
I'd suggest to slightly move the description down in the chapter

File: docs/user_manual/working_with_projections/img/clear_used_crs.png
[the remark this answers is not in the document]
Comment (selmaVH1) [reply docs/user_manual/working_with_projections/img/clear_used_crs.png:1]: @DelazJ I applied suggested changes, you can review it. I'm not sure that indentation is right..

File: docs/user_manual/working_with_projections/img/crs_selector.png
Comment (DelazJ) [inline docs/user_manual/working_with_projections/img/crs_selector.png:1]: @selmaVH1 Do you remember how you get to this dialog? It looks weird to me that it is named "QGIS3".

Comment (selmaVH1) [reply docs/user_manual/working_with_projections/img/crs_selector.png:1]: @DelazJ Layer -> Set CRS of Layer(s), I tried also right-click on layer -> Layer CRS -> Set layer CRS.

Comment (DelazJ) [reply docs/user_manual/working_with_projections/img/crs_selector.png:1]: Indeed. I could reproduce this on 3.28 and 3.32 on macOS and Windows. master seems to display a title though...

**Pull request merged**